src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -108,4 +108,3 @@ $screen-md-max: calc(#{$screen-lg-min} - 1px) !default;
 $screen-lg-max: calc(#{$screen-xl-min} - 1px) !default;
 
 $navbar-height: 43px !default;
-$top-notification-height: 37.6px !default;
```

Comment:
@nizamial09 it'd be nice to check how these changes get along with the downstream branding ;-)